cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -334,10 +337,35 @@ public void deleteNamespace(Id.Namespace namespace) throws Exception {
   }
 
   @Override
+  public void deleteNamespaceData(NamespaceId namespace) throws Exception {
```

Comment:
It seems a bit specific to have this functionality here (also exposed to the user when they're writing their test cases).
Would they ever need to call this?